runtime/src/main/java/io/micronaut/reactive/rxjava2/RxInstrumentedConnectableObservable.java

```diff
@@ -59,8 +59,7 @@ protected void subscribeActual(Observer<? super T> o) {
             instrumenter.beforeInvocation();
             source.subscribe(o);
         } finally {
-            instrumenter.afterInvocation(false);
-            active = false;
```

Comment:
@dstepanov This does not seem right here. Did you want to remove the active field completely or did you just accidentally deleted line 63?

Reply:
You're right, that was a mistake